src/main/java/com/employeesService/EmployeesService/model/Employee.java

```diff
@@ -31,6 +32,7 @@
     @JsonIgnore
     private List<Salary> salaries = new ArrayList<>();
 
+
     public List<Salary> getSalaries() {
```

Comment:
```suggestion
    private List<Salary> salaries = new ArrayList<>();

    public List<Salary> getSalaries() {
```

Always leave one empty line between class members!